<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media OrQanizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStatusBar {
background: url(:/statusBar/background) repeat-x;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="centralWidget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="sideBar" native="true">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget { 
background-color: rgb(60,60,60);
border-right: 1px solid rgb(16,16,16);
}
QToolButton {
padding-top: 4px;
border: none;
color: rgba(106,106,106, 100);
background-image: url(:/sideBar/button_background_nc);
}
QToolButton:pressed {
color: rgba(150,150,150, 255);
background-image: url(:/sideBar/button_background_p);
}
QToolButton:checked, QToolButton:checked:pressed {
color: rgba(150,150,150, 255);
background-image: url(:/sideBar/button_background_c);
}
QWidget#separator {
border-top: 1px solid rgb(38,38,38);
border-bottom: 1px solid rgb(77,77,77);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="buttonDownloads">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>Downloads</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonSeries">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="text">
             <string>TV Shows</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="separator" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>2</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>2</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="opaqueResize">
          <bool>true</bool>
         </property>
         <property name="handleWidth">
          <number>1</number>
         </property>
         <widget class="QStackedWidget" name="centralStackedWidget">
          <property name="styleSheet">
           <string notr="true">QWidget#centralStackedWidget {
background-color: rgb(227,227,227);
}</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="pageDownloads">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTreeView" name="treeViewDownloads"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageSeries">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter_2">
              <property name="midLineWidth">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="opaqueResize">
               <bool>false</bool>
              </property>
              <property name="handleWidth">
               <number>1</number>
              </property>
              <widget class="QTreeView" name="treeViewSeries">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <property name="headerHidden">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QTreeView" name="treeViewSeasons">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <property name="headerHidden">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QTreeView" name="treeViewEpisodes">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <attribute name="headerVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDownloads"/>
    <addaction name="actionTV_Shows"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionDownload_Preferences"/>
    <addaction name="actionPremiumizeMe_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuDownloads">
    <property name="title">
     <string>Downloads</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStopDownloads"/>
    <addaction name="separator"/>
    <addaction name="actionDeleteDownload"/>
    <addaction name="actionResetDownload"/>
    <addaction name="actionExtract"/>
   </widget>
   <widget class="QMenu" name="menuTV_Shows">
    <property name="title">
     <string>TV Shows</string>
    </property>
    <addaction name="actionAdd_show"/>
   </widget>
   <addaction name="menuDownloads"/>
   <addaction name="menuTV_Shows"/>
   <addaction name="menuView"/>
   <addaction name="menuExtras"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>34</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionDownloads">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Downloads</string>
   </property>
   <property name="toolTip">
    <string>Show downloads</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionDownload_Preferences">
   <property name="text">
    <string>Download Preferences...</string>
   </property>
  </action>
  <action name="actionPremiumizeMe_Preferences">
   <property name="text">
    <string>PremiumizeMe Preferences...</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start queue</string>
   </property>
   <property name="toolTip">
    <string>Start Downloads</string>
   </property>
  </action>
  <action name="actionDeleteDownload">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionResetDownload">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionStopDownloads">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop all</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
  <action name="actionTV_Shows">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TV Shows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionAdd_show">
   <property name="text">
    <string>Add show...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
